ts/model_loader.py

```diff
@@ -23,14 +24,8 @@ class ModelLoaderFactory(object):
     """
 
     @staticmethod
-    def get_model_loader(model_dir):
-        manifest_file = os.path.join(model_dir, "MAR-INF/MANIFEST.json")
-        if os.path.exists(manifest_file):
-            return TsModelLoader()
-        elif os.path.exists(os.path.join(model_dir, "MANIFEST.json")):
-            return LegacyModelLoader()
```

Comment:
Not sure if this is a lint rule we set up.
Not clear why this isn't necessary. Could you please explain?

Reply:
The LegacyModelLoader was ported from MMS and is no more in use in TorchServe.